src/cloud/reference/cli-ref-topic.md

```diff
@@ -5,30 +5,65 @@ functional_areas:
   - Cloud
 ---
 
-The Magento Cloud CLI is the utility for managing Magento Cloud projects and environments, which implements many features of the Project Web Interface and more. This utility provides an advanced management interface for developers, DevOps and system administrators, for doing routines and automation tasks. You can install and use this utility for all {{site.data.var.ece}} Starter and Pro Integration environments. You cannot use these commands for the Pro Staging and Production environments which aren't listed in the cloud UI.
+The Magento Cloud CLI is the utility for managing Magento Cloud projects and environments, which implements many features of the Project Web Interface and more. This utility provides an advanced management interface for developers and system administrators to perform routines and automation tasks. You can install and use this utility for all {{site.data.var.ece}} Starter and Pro Integration environments.
 
-You can install the Magento Cloud CLI when setting up your local environment for development:
+{:.procedure}
+To install the Magento Cloud CLI:
 
-*  [Install Magento prerequisites]({{ site.baseurl }}/cloud/before/before-workspace-magento-prereqs.html)
-*  [Enable SSH keys]({{ site.baseurl }}/cloud/before/before-workspace-ssh.html)
+1. On your local workstation, change to a directory where you plan to clone the cloud projects and where the [Magento file system owner](https://glossary.magento.com/magento-file-system-owner) has _write_ access.
 
-## Common commands
+1. Install the `magento-cloud` CLI.
 
-The following lists the available `magento-cloud` CLI commands:
+   ```bash
+   curl -sS https://accounts.magento.cloud/cli/installer | php
+   ```
 
-```bash
-magento-cloud list
-```
+1. Add `magento-cloud` CLI to the bash profile.
+
+   ```bash
+   export PATH=$PATH:$HOME/.magento-cloud/bin
+   ```
+
+   Consult operating system documentation for details.
+
+1. Source the update bash profile.
```

Comment:
I've always found "source the bash profile obscure--you run the `source` command to reload the updated profile.

```suggestion
1. Reload the updated bash profile.
```